src/calculateRentalCost.js

```diff
@@ -25,10 +25,24 @@
  * @param {number} days
  *
  * @return {number}
+ *
  */
+const pay = 40;
+const FullDiscaunt = 50;
+const GetOffTotal = 20;
```

Comment:
general variable name must start with small letter.
```suggestion
const fullDiscount = 50;
const partialDiscount = 20;
```